starfish/core/types/_spot_finding_resutls.py

```diff
@@ -0,0 +1,88 @@
+from typing import List, Mapping, MutableMapping, Optional, Tuple
+
+from starfish.core.types import Axes, SpotAttributes
+
+AXES_ORDER = (Axes.ROUND, Axes.CH)
+
+
+class SpotFindingResults:
```

Comment:
Couple design questions:

1. Is it ever the case we would want to do spot finding on a zplane basis?
2. Why not just pass in round, ch as arguments to the methods in this class?

Reply:
I think tony and I resolved these in person, but tony yell at me if not